### ISSUE TITLE
Deleting multiple rows from a table fails

### ISSUE THREAD

**Issue by jgeewax** [GoogleCloudPlatform/google-cloud-node#2098]
What I want to do:

```js
table.deleteRows(['1', '2', '3']);
```

(My ID field is an `INT64` type, but I get that I have to use strings here because 64-bit integers don't nicely fit in Node's world....)

What happens:

```
table.deleteRows(['1', '2', '3'])
Promise { <pending> }
> (node:111838) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 10): Error: Wrong number of key parts for mytable. Expected: 1. Got: ["1","2","3"]
```

To check that I'm not crazy, the following works as expected.

```js
table.deleteRows(['1']);
```

I'm guessing that this has to do with the hierarchical key stuff, and that what's actually happening here is the list is a *single* key, which has many parts -- not a list of keys.

To double check, I tried doing each as their own list, which similarly failed:

```js
table.deleteRows([ ['1'], ['2'], ['3'] ]);
```

**Comment by vkedia** [comment]
@callmehiphop @stephenplusplus  Is this behavior expected? I would have thought the last snippet there would have worked. How else would one delete multiple rows?

**Comment by stephenplusplus** [comment]
@jgeewax I believe I indirectly solved this one in #2141 -- feel like cloning and double checking? Or you can see if the new tests in the system tests covers your use case.
